core/app/mailers/spree/carton_mailer.rb

```diff
@@ -2,11 +2,13 @@ module Spree
   class CartonMailer < BaseMailer
     # Send an email to customers to notify that an individual carton has been
     # shipped.
-    def shipped_email(carton_id, resend: false)
-      @carton = Spree::Carton.find(carton_id)
+    def shipped_email(order, carton, resend: false)
```

Comment:
Should we consider adding this as a new method, and deprecating the old `shipped_email(carton_id, resend: false)` in order to maintain API compatibility?